org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/healthcheck/AbstractHealthCheckServlet.java

```diff
@@ -118,16 +132,38 @@ protected void doChecks(RunContext context, HttpServletRequest req, HttpServletR
     }
   }
 
-  protected List<IHealthChecker> getActiveHealthCheckers() {
+  /**
+   * Extract the category (if any) from the request.
+   *
+   * @return <code>null</code> if no (valid?) category was provided
+   */
+  protected HealthCheckCategoryId parseCategory(HttpServletRequest req) {
+    String inputCategory = StringUtility.nullIfEmpty(req.getParameter(QUERY_PARAMETER_NAME_CATEGORY));
+    HealthCheckCategoryId category = null;
+    if (inputCategory != null) {
+      category = BEANS.all(IHealthCheckCategoryProvider.class)
```

Comment:
Use BEANS.all(IHealthCheckCategory.class)

Reply:
Fixed in 184fa26d2119186562032245b73ffa38f35d5b9e